### ISSUE TITLE
Tab order of pane quadrants doesn't match visual layout

### ISSUE THREAD

**Issue by gtritchie** [rstudio/rstudio#5980]
### System details

    RStudio Edition : Desktop or Server
    RStudio Version : 1.3.723
    OS Version      : any
    R Version       : any

### Steps to reproduce the problem
- use default pane layout (editor upper left, console lower left, etc.)
- get keyboard focus to the start of the page (e.g. the menu on server, toolbar on desktop)
- start tabbing through the page to get beyond the menu/toolbars

### Describe the problem in detail
- after the toolbar, focus goes to the upper-right quadrant ("TabSet1"), then lower-right ("TabSet2"), then the editor pane (upper-left), and finally the console pane (lower-left)
 
### Describe the behavior you expected
- this is unexpected for a keyboard user who can also see what's going on and would expect more of a left-to-right, top-to-bottom flow
- potentially a violation of https://www.w3.org/TR/WCAG21/#focus-order
- not a showstopper for actual use by a blind user since the "physical" pane order and layout isn't a particularly important factor (in fact, arguably, the existence of the 4 quadrants is a hinderance more than a help)
